src/Service/Sync.php

```diff
@@ -87,6 +130,92 @@ private function makeRequest($client, $body)
         return json_decode($response->getBody()->getContents(), true);
     }
 
+    private function displayUnusedSegments($responseData, $command, $showPurgeable, $purge) {
+        $unusedSegments = collect($responseData['unused_segments']);
+
+        $yamlUnusedSegments    = $unusedSegments->filter(function ($unusedSegment) {
```

Comment:
Its very uncommon to align assignments like this in php.
Actually you can run " vendor/bin/phpcbf --standard=psr2 src/", thats probably simplest way to get "most common" formating